### ISSUE TITLE
Unselecting behaves inconsistent on gone packages

### ISSUE THREAD

**Issue by coolo** [openSUSE/open-build-service#7377]
If I remove the package manually in the staging project and unselect afterwards, I get an error 400 but the package is unselected in the DB:

```
DELETE https://api-test.opensuse.org/staging/openSUSE:Factory/staging_projects/openSUSE:Factory:Staging:A/staged_requests
Server returned an error: HTTP Error 400: Bad Request

<status code="invalid_request">
  <summary>Error while unassigning requests: Could not delete packages gnutls.</summary>
</status>
```

But afterwards gnutls is no longer tracked in A.

**Comment by eduardoj** [comment]
This was fixed in #8460. Closing.

**Comment by coolo** [comment]
confirmed on build-test
